### DISCUSSION TITLE
Discordでチャットのお願いと情報系の問い合わせ

### DISCUSSION THREAD

**Discussion by emihumen33** [Seasawher/Seasawher#8] [Q&A]
外国の大学に卒業から、RIMSの入試に関する問題は沢山あります！お願いします🙇
Discordのフレンド追加請求を送信しました、わたしのアカウントはemihumen、名前はJ.JDです。
わたしも数学における情報科学の分野に興味があります。特には計算機で証明、leanやcoqなど。
でも、過去問の情報系の問題を見ました後、これは何？？？？という感じが生まれる。だから、情報系に合格するために何(参考書)を勉強すべきですか、seasawherさんがrimsの情報系の知り合いがありますかという質問があります。本当にありがとうございました😭🙏

**Comment by Seasawher** [comment]
@emihumen33 

通知が来ていなくて、気づいていませんでした。失礼しました。
おそらく既に解答済みだと思うので、クローズします。
